cclib/parser/molcasparser.py

```diff
@@ -725,6 +725,59 @@ def extract(self, inputfile, line):
                     self.ccenergies= []
                 self.ccenergies.append(ccenergies)
 
+        #  ++    Primitive basis info:
+        #        ---------------------
+        #
+        #
+        #                      *****************************************************
+        #                      ******** Primitive Basis Functions (Valence) ********
+        #                      *****************************************************
+        #
+        #
+        #   Basis set:C.AUG-CC-PVQZ.........                                                          
+        #
+        #                    Type         
+        #                     s
+        #             No.      Exponent    Contraction Coefficients
+        #             1  0.339800000D+05   0.000091  -0.000019   0.000000   0.000000   0.000000   0.000000
+        #             2  0.508900000D+04   0.000704  -0.000151   0.000000   0.000000   0.000000   0.000000
+        # ...
+        # ...
+        #             29  0.424000000D+00   0.000000   1.000000
+        #
+        #   Number of primitives                                   93
+        #   Number of basis functions                              80
+        #
+        #  --
+        if '++    Primitive basis info:' in line:
+            self.skip_lines(inputfile, ['d', 'b', 'b', 's', 'header', 's', 'b'])
+            line = next(inputfile)
+            while '--' not in line and '****' not in line:
+                if 'Basis set:' in line:
+                    try:
+                        self.append_attribute('gbasis', gbasis)    
+                    except:
+                        pass
+                    gbasis = []
+                    self.skip_line(inputfile, 'b')
+                    line = next(inputfile)
+                    while 'Type' in line:
+                        coeffs = []
+                        line = next(inputfile)
+                        bftype = line.split()[0].upper()
+                        self.skip_line(inputfile, 'headers')
+                        line = next(inputfile)
+                        while line.split():
+                            coeffs.append((self.float(line.split()[1]), self.float(line.split()[2])))
```

Comment:
Unfortunately, I think you need the other coeffs depending on their values. How does the result here compare with the other parsers?

Reply:
I mentioned that in the comment above. That is what I am puzzled with as well.

Reply:
+1 Please use the other columns and we'll see how things stand.